lib/commands/run.ts

```diff
@@ -53,7 +53,7 @@ export class RunAndroidCommand extends RunCommandBase implements ICommand {
 			if (this.$options.release && (!this.$options.keyStorePath || !this.$options.keyStorePassword || !this.$options.keyStoreAlias || !this.$options.keyStoreAliasPassword)) {
 				this.$errors.fail("When producing a release build, you need to specify all --key-store-* options.");
 			}
-			return args.length === 0 && this.$platformService.validateOptions(this.$platformsData.availablePlatforms.Android).wait();
+			return this.$platformService.validateOptions(this.$platformsData.availablePlatforms.Android).wait();
```

Comment:
In case user writes `tns run android some other arguments` the command will still be executed, which is change from the current behavior. Is this intended?